fluentlenium-core/src/main/java/org/fluentlenium/core/annotation/Page.java

```diff
@@ -6,6 +6,10 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * @deprecated in favor of {@link javax.inject.Inject}.
+ */
+@Deprecated
```

Comment:
Currently it's marked deprecated, but works just as ```@Inject``` annotation. I don't plan to remove it for 1.0 release, but maybe for 2.0 (which will be Java8 only).